src/mongocxx/include/mongocxx/v_noabi/mongocxx/client_session.hpp

```diff
@@ -185,10 +185,18 @@ class client_session {
     /// the transaction, the entire sequence may be retried, and the callback
     /// may be run multiple times.
     ///
-    /// If the user callback calls driver methods that run operations against the
-    /// server that can throw an operation_exception (ex: collection::insert_one),
-    /// the user callback should allow those exceptions to propagate up the stack
-    /// so they can be caught and processed by the with_transaction helper.
+    /// This method has an internal non-adjustable time limit of 120 seconds,
+    /// including all retries.
+    ///
+    /// If the user callback invokes driver methods that run operations against the
+    /// server which could throw an operation_exception, the user callback MUST allow
+    /// those exceptions to propagate up the stack so they can be caught and processed
+    /// by the with_transaction() helper.
+    ///
+    /// For example, a callback that invokes collection::insert_one may encounter a
+    /// "DuplicateKey" error with accompanying server-side transaction abort. If this
```

Comment:
```suggestion
    /// "duplicate key" error with accompanying server-side transaction abort. If this
```

Suggest revising to match server error message:

```
> db.runCommand({"insert": "foo", "documents": [{_id: 1}, {_id: 1}]})
{
  n: 1,
  writeErrors: [
    {
      index: 1,
      code: 11000,
      errmsg: 'E11000 duplicate key error collection: test.foo index: _id_ dup key: { : 1 }'
    }
  ],
  opTime: { ts: Timestamp({ t: 1730216400, i: 4 }), t: Long('2') },
  electionId: ObjectId('7fffffff0000000000000002'),
  ok: 1,
  operationTime: Timestamp({ t: 1730216400, i: 4 }),
  '$clusterTime': {
    clusterTime: Timestamp({ t: 1730216400, i: 5 }),
    signature: {
      hash: Binary.createFromBase64('AAAAAAAAAAAAAAAAAAAAAAAAAAA=', 0),
      keyId: Long('0')
    }
  }
}
```

Reply:
Got it, thanks. I saw the "DuplicateKey" terminology more frequently in libmongoc but I guess that's actually just localized to the json test runner.